bin/dump-boring-reads.py

```diff
@@ -30,26 +43,39 @@ def main(args):
     with open(args.fasta, 'r') as seqfile:
         seqs = {record.id: record for record in SeqIO.parse(seqfile, 'fasta')}
 
+    if args.genomemask:
+        genomemask = khmer.Countgraph(args.mask_k, int(args.maskmemory / 4), 4)
+        genomemask.consume_fasta(args.genomemask)
+
     bam = pysam.AlignmentFile(args.bam, 'rb')
     for i, record in enumerate(bam):
         if (i+1) % 10000 == 0:
             print('...processed', i, 'records', file=sys.stderr)
 
-        if args.seqid:
-            if record.reference_id < 0 or bam.get_reference_name(record.reference_id) != args.seqid:
-                continue
-
         if record.is_secondary or record.is_supplementary:
             continue
 
-        matchcigar = '{:d}M'.format(record.rlen)
-        if record.cigarstring != matchcigar:
```

Comment:
Previously, only reads with 126M CIGAR strings were kept. This was a bug. The intention was to restrict the sequence identity check to reads with 126M (if CIGAR is not 126M then we always want to keep it). This surely explains why no indels have yet been identified.